Add plugin: Occura

Krusty84 opened the pull request:
# I am submitting a new Community Plugin

## Repo URL

<!--- Paste a link to your repo here for easy access -->
Link to my plugin:

## Release Checklist
- [X] I have tested the plugin on
  - [X]  Windows
  - [X]  macOS
  - [ ]  Linux
  - [ ]  Android _(if applicable)_
  - [ ]  iOS _(if applicable)_
- [X] My GitHub release contains all required files (as individual files, not just in the source.zip / source.tar.gz)
  - [X] `main.js`
  - [X] `manifest.json`
  - [X] `styles.css` _(optional)_
- [X] GitHub release name matches the exact version number specified in my manifest.json (_**Note:** Use the exact version number, don't include a prefix `v`_)
- [X] The `id` in my `manifest.json` matches the `id` in the `community-plugins.json` file.
- [X] My README.md describes the plugin's purpose and provides clear usage instructions.
- [X] I have read the developer policies at https://docs.obsidian.md/Developer+policies, and have assessed my plugins's adherence to these policies.
- [X] I have read the tips in https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines and have self-reviewed my plugin to avoid these common pitfalls.
- [X] I have added a license in the LICENSE file.
- [X] My project respects and is compatible with the original license of any code from other plugins that I'm using.
      I have given proper attribution to these other projects in my `README.md`.

joethei commented:
["id": "occura-09102024",](https://github.com/Krusty84/obsidian-occura-plugin/blob/cc8ecd44c737044f4d402ee4afc650b76944bdec/manifest.json#L2)
Remove these random numbers from the ID.

["description": "A plugin that helps find all occurrences of selected text in notes, similar to Notepad++ or any IDE.",](https://github.com/Krusty84/obsidian-occura-plugin/blob/cc8ecd44c737044f4d402ee4afc650b76944bdec/manifest.json#L6)
Avoid including sentences like "This is a plugin that does ..." in your description, it should be self-evident that this is a plugin for Obsidian.
We have published a guide for plugin descriptions to follow here: https://docs.obsidian.md/Plugins/Releasing/Submission+requirements+for+plugins#Keep+plugin+descriptions+short+and+simple.

[This CSS file will be included with your plugin, and](https://github.com/Krusty84/obsidian-occura-plugin/blob/cc8ecd44c737044f4d402ee4afc650b76944bdec/styles.css#L3-L6)
Remove this comment.

[name: 'Toggle Highlight Occurrences',](https://github.com/Krusty84/obsidian-occura-plugin/blob/cc8ecd44c737044f4d402ee4afc650b76944bdec/main.ts#L36), [iconEl.setAttribute('aria-label', 'Disable Highlighting');](https://github.com/Krusty84/obsidian-occura-plugin/blob/cc8ecd44c737044f4d402ee4afc650b76944bdec/main.ts#L200), [iconEl.setAttribute('aria-label', 'Enable Highlighting');](https://github.com/Krusty84/obsidian-occura-plugin/blob/cc8ecd44c737044f4d402ee4afc650b76944bdec/main.ts#L207)
[Use sentence case in UI](https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines#Use+sentence+case+in+UI)

[iconEl.setAttribute('aria-label', 'Disable Highlighting');](https://github.com/Krusty84/obsidian-occura-plugin/blob/cc8ecd44c737044f4d402ee4afc650b76944bdec/main.ts#L200), [iconEl.setAttribute('aria-label', 'Enable Highlighting');](https://github.com/Krusty84/obsidian-occura-plugin/blob/cc8ecd44c737044f4d402ee4afc650b76944bdec/main.ts#L207)
use the `setTooltip` function.

Krusty84 commented:
Hello Dear @goethe Johannes, 
I have fixed it in my latest commit, and I have changed files (main.js, manifest.json and styles.css) in plugin release.
Danke Schon for comments and advice's!
